public-ui/src/app/components/db-no-results/db-no-results.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Input, OnChanges, OnDestroy, Output } from '@angular/core';
+import { Router } from '@angular/router';
+import { ApiModule, DataBrowserService, DomainInfosAndSurveyModulesResponse } from 'publicGenerated';
```

Comment:
Nit: remove unused imports `ApiModule` and `DomainInfosAndSurveyModulesResponse`